editor/src/components/canvas/controls/formula-bar.tsx

```diff
@@ -166,7 +166,7 @@ export const FormulaBar = betterReactMemo('FormulaBar', () => {
           }}
           onChange={onInputChange}
           onBlur={onBlur}
-          value={simpleText}
+          value={simpleText.trimStart()}
```

Comment:
This should probably be using `.trim()` rather than `.trimStart()`

Reply:
Haha, that was my thinking too. Until I tried typing `Hello World` and couldn't get the space in :D

Reply:
Ha of course!

Reply:
I've tested this out, and it gives a rather unsavoury result when the text is on a separate line to the element, e.g.:
```
    <div
      style={{
        width: '100%',
        height: '100%',
        backgroundColor: '#FFFFFF',
        position: 'relative',
      }}
    >
      Hello World
    </div>
```
when making a non-change via the omnibar will result in:
```
    <div
      style={{
        width: '100%',
        height: '100%',
        backgroundColor: '#FFFFFF',
        position: 'relative',
      }}
    >
      Hello World{' '}
    </div>
```

I've checked and you can fix that by using `trim()` when setting the value on blur, but I've also noticed another issue in the way we're setting the initial value of the omnibar which potentially will result in data loss when typing fast. I'm going to create an issue ticket for it.

Reply:
I've created https://github.com/concrete-utopia/utopia/issues/1646 for this